app/src/processing/app/Base.java

```diff
@@ -1617,13 +1667,13 @@ private static boolean ifThereAreVisibleItemsOn(JMenu menu) {
     return false;
   }
 
-  private JMenu getBoardCustomMenu(String label) throws Exception {
+  private JMenu getBoardCustomMenu(String label) {
     for (JMenu menu : boardsCustomMenus) {
       if (label.equals(menu.getText())) {
         return menu;
       }
     }
-    throw new Exception("Custom menu not found!");
+    return null;
```

Comment:
Why this change? Is this case suddenly a normal occurence with this LRU list? Or is this just an unrelated improvement (that would perhaps be better in a separate commit)?

Reply:
It's indeed a separate commit, thanks